deployments/deployments.json

```diff
@@ -203,29 +204,29 @@
   "41455": {
     "SpokePool": { "address": "0x13fDac9F9b4777705db45291bbFF3c972c6d1d97", "blockNumber": 4240318 },
     "SpokePoolVerifier": { "address": "0x3Fb9cED51E968594C87963a371Ed90c39519f65A", "blockNumber": 8669066 },
-    "MulticallHandler": { "address": "0x924a9f036260DdD5808007E1AA95f08eD08aA569", "blockNumber": 4112529 }
+    "MulticallHandler": { "address": "0x0F7Ae28dE1C8532170AD4ee566B5801485c13a0E", "blockNumber": 4112529 }
```

Comment:
We can delete Aleph Zero - the chain isn't up any longer and the RPCs are starting to disappear.

Reply:
https://github.com/across-protocol/contracts/pull/1058/commits/a8b178a1d65649c4e6ec18b602f0ce90e324b775